environment.py

```diff
@@ -67,12 +70,6 @@ def get_reward(self, action, date=None, index=None):
         #         returns = row.iloc[1:]
         returns = self.get_continuous_state(date=date, index=index)
         portfolio_return = np.dot(action, returns)
+        sharpe_ratio = (portfolio_return - 0.02) / (np.std(returns) if np.std(returns) > 0 else 1)
         # I just found out for the continuous agent, the reward is simply the weighted sum
-        return portfolio_return
-        log_return = np.log(1 + portfolio_return)
-
-        # assume risk-free rate of 0.02 for Sharpe ratio calculation
-        # sharpe_ratio = (log_return - 0.02) / (np.std(returns) if np.std(returns) > 0 else 1)
-        # reward = log_return + sharpe_ratio  # TODO - POC only, calculations to be finalized later
-        reward = log_return
-        return reward
+        return sharpe_ratio if self.use_sharpe_ratio_reward else portfolio_return
```

Comment:
Please use the differential sharpe ratio as we are calculating the one-step sharpe ratio.
Reference: https://github.com/AchillesJJ/DSR

Reply:
just updated this using the methods referenced in the paper, as well as the reference you provided. please help me verify the correctness of this updated approach, thanks!